java/ql/test/query-tests/security/CWE-287/InsecureKeys/Test2/Test.java

```diff
@@ -0,0 +1,13 @@
+import android.security.keystore.KeyGenParameterSpec;
+import android.hardware.biometrics.BiometricPrompt;
+import android.security.keystore.KeyProperties;
+
+class Test {
```

Comment:
I'd like to see the GOOD example showed in the QHelp as a test case here too, to make sure that we work as intended on the code samples we give to users.